mocks/timestamp.js

```diff
@@ -19,34 +19,34 @@ class Timestamp {
   }
 
   toDate() {
-    const d = new Date(0);
-    d.setTime(this.seconds * 1000);
-    d.setMilliseconds(this.nanoseconds / 1000000);
-    return mockTimestampToDate(...arguments) || d;
+    return mockTimestampToDate(...arguments) || new Date(this._toMillis());
```

Comment:
Not sure if this has already been addressed (I think someone may have logged an issue about this?) but when I wrote this PR the conversion Date => Timestamp => Date did not work correctly

Reply:
https://github.com/Upstatement/firestore-jest-mock/issues/94

Reply:
yeah, that solution is pretty nice (although you still need this class to call the mocks).  /shrugs, either/or solution works